src/Template/Element/Form/relation.twig

```diff
@@ -108,15 +108,13 @@
         {{ Form.control(relationName ~ 'removeRelated', {
             'type': 'hidden',
             'name': 'relations[' ~ relationName ~ '][removeRelated]',
-            'v-model': 'removedRelationsData'
```

Comment:
Why are we removing `'v-model': 'removedRelationsData'` and `'v-model': 'addedRelationsData'` ?
It affects object relations handling in general.

Also in `src/Template/Layout/js/app/components/relation-view/relation-view.js`